tic_tac_toe/src/board.py

```diff
@@ -0,0 +1,203 @@
+# Board Representation
```

Comment:
What's this and, more importantly, why is it not a description of the Board class?